src/ex6/client/src/components/todo-element.js

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from "react";
+import { deleteItem, markItemAsOld, changeItemStatus } from "../item_client.js";
+import PropTypes from 'prop-types';
+
+export function TodoElement(props) {
+  const [todoClassName, setTodoClassName] = useState("todo-li existing-todo")
```

Comment:
className is usually not the state but the state affect it, so in this case the animation progress will be the state

Reply:
fixed